main.py

```diff
@@ -7,17 +7,20 @@
 from datetime import datetime
 
 
+
+# This function is responsible for calculating a date in range start to end.

```

Comment:
1. As commented in "this is the way" - fix the documentation.
2. Give better names to your parameters (instead of just start and end).